### PR TITLE
DCM-47: Initiate the location mapping user interface

### PR DESCRIPTION
**Pull request by Piumal1999**
## The issue I worked on
See [https://issues.openmrs.org/browse/DCM-47](https://issues.openmrs.org/browse/DCM-47) 

## Description of what I changed:
Created the User interface of location mapping
This UI displays the mapped location-orgUnits which are saved in the database
Note- Saving/Updating feature will be added under a new JIRA issue

Screenshot 2021-06-23 at 00 32 15

### CODE REVIEW

File: omod/src/main/webapp/template/localHeader.jsp [hunk 1]
```diff
@@ -10,7 +10,14 @@
...
 	</li>
-	
+
+	<li
+			<c:if test='<%= request.getRequestURI().contains("/locationMapping") %>'>class="active"</c:if>>
+		<a
+				href="${pageContext.request.contextPath}/module/dhisconnector/locationMapping.form"><spring:message
+				code="dhisconnector.locationMapping" /></a>
+	</li>
+
```
Comment (jayasanka-sack) [inline omod/src/main/webapp/template/localHeader.jsp:20]: Shall we fix the formattings of this section?

File: omod/src/main/webapp/locationMapping.jsp [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        <tr>
+            <th><spring:message code="dhisconnector.locationMapping.location"/></th>
+            <th><spring:message code="dhisconnector.locationMapping.orgUnit"/></th>
+        </tr>
+        </thead>
+
+        <tbody>
+        <c:forEach items="${locations}" var="location">
+            <tr class="evenRow">
+
```
Comment (jayasanka-sack) [inline omod/src/main/webapp/locationMapping.jsp:27]: Let's temove unwanted spaces.

Comment (Piumal1999) [reply omod/src/main/webapp/locationMapping.jsp:27]: Okay. I'll update the PR.

File: omod/src/main/webapp/template/localHeader.jsp (hunk 1 above)
Comment (Piumal1999) [reply omod/src/main/webapp/template/localHeader.jsp:20]: Alright. I'll update the PR.

Comment (Piumal1999) [reply omod/src/main/webapp/template/localHeader.jsp]: Fixed with [041fd7b](https://github.com/openmrs/openmrs-module-dhisconnector/commit/041fd7b82d787717170bb3a0acb07c859cc76fd9)

File: omod/src/main/webapp/locationMapping.jsp (hunk 2 above)
Comment (Piumal1999) [reply omod/src/main/webapp/locationMapping.jsp]: Fixed with [041fd7b](https://github.com/openmrs/openmrs-module-dhisconnector/commit/041fd7b82d787717170bb3a0acb07c859cc76fd9)

**Review by jayasanka-sack — APPROVED** [review]
Good work @Piumal1999!

**Pull request merged**